src/components/products/Products.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import '../style/productStyle.css'
+
+function Products () {
+
+    return (
+        <div className="ProductWrapper">
```

Comment:
For styles let's use SASS, in your case, it has to be productStyle.scss
for className let's use "-" or "_" to separate words, not camelCase, camelCase we usually use for variables names
for example className="product-wrapper"